docker-compose.yml

```diff
@@ -79,7 +79,7 @@ services:
     env_file: ./frontend/.env
     environment:
       - NODE_ENV=development
-      - KEYCLOAK_URL=http://host.docker.internal:8080/auth/admin/realms/quartech/users
+      - API_URL=http://host.docker.internal:5000
```

Comment:
The **API_URL** was hardcoded in `setupProxy.js`. I made it an env var to support the use case of developing locally on the frontend while targeting a remote API; e.g. pims-dev.pathfinder.gov.bc.ca/api